content/legal/privacy-notice.md

```diff
@@ -1,6 +1,11 @@
 +++
 title = "Matrix.org Foundation Privacy Policy"
-version = "3.1.0"
+version = "3.2.0"
```

Comment:
we should probably adjust the template for these versioned pages (probably https://github.com/matrix-org/matrix.org/blob/main/templates/legal.html) so they actually show the version to the reader.